deploy/crds/nmstate.io_nmstates.yaml

```diff
@@ -89,6 +89,58 @@ spec:
                       type: string
                   type: object
                 type: array
+              webhookNodeSelector:
+                additionalProperties:
+                  type: string
+                description: InfraNodeSelector is an optional selector that will be
+                  added to webhook & certmanager Deployment manifests If InfraNodeSelector
+                  is specified, the webhook and certmanager will run only on nodes
+                  that have each of the indicated key-value pairs as labels applied
+                  to the node.
+                type: object
+              webhookTolerations:
```

Comment:
This CRD has to be re-generated is still pointing to "webhook"

Reply:
I regenerated it and push the changes.